torchvision/prototype/transforms/_augment.py

```diff
@@ -112,6 +119,8 @@ def __init__(self, *, alpha: float) -> None:
         self._dist = torch.distributions.Beta(torch.tensor([alpha]), torch.tensor([alpha]))
 
     def _get_params(self, sample: Any) -> Dict[str, Any]:
+        # Question: Shall we enforce here the existence of Labels in the sample? If yes, this method of validating
+        # input won't work if get_params() gets public and the user sis able to provide params in forward.
```

Comment:
I agree we should be loud about this, but I don't know about an error. Maybe a warning is sufficient? It is safer to upgrade to an error later than downgrading an error to a warning.

--- 

For input validation like this, we should simply do

```py
def forward(self, *inputs):
    validate_inputs(inputs)
    return super().forward(*inputs)
```

Reply:
Because 99% of the classes don't need to validate the output, I would probably avoid creating a mechanism that would need to be overwritten etc (like a proper method). The solution you propose of placing this on top of forward sounds good to me. I would only put those validations straight in `forward`. I would like to hear other opinions thought.

We should figure out these matters prior starting adding the remaining classes.